app/models/taxon_facet.rb

```diff
@@ -43,8 +43,25 @@ def query_params
     }
   end
 
+  def selected_taxon_value
+    selected_level_one_value || selected_level_two_value || selected_topic_value
+  end
+
 private
 
+  def selected_topic_value
+    return unless @value_hash["topic"]
+
+    content_id = ContentItem.from_content_store(@value_hash["topic"]).content_id
```

Comment:
I see why this is necessary, because the incoming param is a base path, and we need a Content ID. There are a couple of downsides to this:

- We have to do the extra API call
- It's inconsistent with the other params, which are Content IDs

Since we're going to have to repoint all the existing links anyway, how about we assume the incoming param is a Content ID? Then we can avoid doing a lookup.

Reply:
(I think this was another one of those implicit assumptions in my head. Sorry!)